README.md

```diff
@@ -43,6 +43,8 @@ Current stable release is able to persist Orion context data in:
 * [MongoDB](https://www.mongodb.org/), the NoSQL document-oriented database.
 * [STH](https://github.com/telefonicaid/IoT-STH), a Short-Term Historic database built on top of MongoDB.
 
+You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) before going deep into the details.
```

Comment:
Some day we should consume a whole sprint reviewing the grammar of all the documents at telefonicaid's repos... any candidate? ;p